dnase/callsnpsMerge.sh

```diff
@@ -163,6 +164,41 @@ bcftools index ${sampleOutdir}/${name}.${mappedgenome}.filtered.vcf.gz
 tabix -p vcf ${sampleOutdir}/${name}.${mappedgenome}.filtered.vcf.gz
 rm -f ${fltvcffiles}
 
+echo
+echo "Run Delly"
+date
+
+## Make a place for the temp bam files with qcfail turned off.
+bam_output_qcOK=${TMPDIR}/${name}.${mappedgenome}.QC_OK_bamfile.bam
+
+## Turn off all qcfail flags.
+python ${src}/changeFlags.py ${sampleOutdir}/${name}.${mappedgenome}.bam ${bam_output_qcOK}
+
+samtools index ${bam_output_qcOK}
+
+set +e
```

Comment:
can you add a comment explaining why set +e is necessary? is it needed for the whole block, or just DELLY? It should encapsulate only the minimum